libkbfs/journal_md_ops_test.go

```diff
@@ -0,0 +1,209 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/keybase/client/go/protocol"
+	"github.com/stretchr/testify/require"
+
+	"golang.org/x/net/context"
+)
+
+// TODO: Clean up the test below.
+
+func TestJournalMDOpsBasics(t *testing.T) {
+	// setup
+	tempdir, err := ioutil.TempDir(os.TempDir(), "journal_md_ops")
+	require.NoError(t, err)
+	defer func() {
+		err := os.RemoveAll(tempdir)
+		require.NoError(t, err)
+	}()
+
+	config := MakeTestConfigOrBust(t, "test_user")
+	defer config.Shutdown()
```

Comment:
Why not `CheckConfigAndShutdown()`?  Are there expected errors?   If so, a comment explaining would be good.